mitmproxy/tools/console/master.py

```diff
@@ -33,6 +33,13 @@
 T = TypeVar("T", str, bytes)
 
 
+def try_unlink(name: str) -> None:
+    try:
+        os.unlink(name)
+    except Exception:
```

Comment:
Thought about whether this should only catch a `FileNotFoundError` so as not to hide other errors. Linting requires the except not be bare at minimum so put `Exception` for now

Reply:
FileNotFoundError makes mose sense here I think.